test/recurring/constants.js

```diff
@@ -6,6 +6,12 @@ const INTERVAL = 'MONTH';
 const INTERVAL_COUNT = 1;
 const PAYMENT_ID = '5e0577bdf4d38b20d542009b';
 const START_DATE = '2020-01-31T17:00:00.000Z';
+const CUSTOMER = {
+  given_names: 'stan',
+  email: 'stanley@xendit.co',
```

Comment:
@bob-xendit can we put dummy email instead?

Reply:
@xen-HendryZheng done change the email to dummy